fq_utils.c

```diff
@@ -457,6 +557,80 @@ fq_debug_stacktrace(fq_debug_bits_t b, const char *tag, int start, int end) {
   if(btname) free(btname);
 }
 
+int fq_serialize(struct iovec **vecs, int *vec_count, uint32_t peermode, size_t off, fq_msg *m) 
```

Comment:
This function is unfinished but also uncalled from anywhere.. Eventually, this will be paired with a `deserialize` to reduce the amount of data we have to write to disk based queues.